src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -302,6 +339,10 @@ define(function (require, exports, module) {
     ColorEditor.prototype._normalizeColorString = function (color) {
         var normalizedColor = color;
 
+        // Convert from 0x notation into hex format string
+        if (color.match(/^0x[0-9a-fA-F]{6}/)) {
```

Comment:
you can re-use the `ensureHexFormat` function I mentioned in the comment before here?